src/routes/jira/jira-atlassian-connect-get.ts

```diff
@@ -90,6 +90,15 @@ const modules = {
 			url: "/jira/app-creation",
 			location: "none",
 			conditions: adminCondition
+		},
+		{
+			key: "github-manual-app-page",
```

Comment:
We shouldn't have the word 'manual' in this path at all as this editing page is for any GHES app regardless of how it was created. If a customer automatically created an app but rotates their GH client secret, they are going to land on this page when they click the 'Edit' option from the dropdown. We are simply reusing the form but it has nothing to do with 'manual' in this part of the flow. 

- [ ] remove all references to the word 'manual' and replace with 'edit'

Reply:
Updated the names from manual to edit